pkg/config/constants/constants.go

```diff
@@ -93,8 +93,8 @@ const (
 	// DefaultServiceAccountName is the default service account to use for remote cluster access.
 	DefaultServiceAccountName = "istio-reader-service-account"
 
-	// DefaultConfigServiceAccountName is the default service account to use for external Istiod cluster access.
-	DefaultConfigServiceAccountName = "istiod-service-account"
+	// DefaultConfigServiceAccountName is the default service account to use for external Istiod config cluster access.
+	DefaultConfigServiceAccountName = "istiod"
```

Comment:
isn't this service account name revisioned?

Reply:
It appends the revision if there is one:
```
name: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
```
But we install the external istiod without a revision, so it's just `istiod` in the initial setup. Is that not the intended name?